Header header

bool commandOnOff
bool onOff

int8 ctrlNb

bool path
int8 pathNb

int32 maxThrust
float64 GainCP
bool noInt

geometry_msgs/Pose deltaPose
